src/modules/sharing/sharing.repository.ts

```diff
@@ -283,14 +312,8 @@ export class SequelizeSharingRepository implements SharingRepository {
           'encryptionKey',
         ],
         [sequelize.literal(`MAX("SharingModel"."created_at")`), 'createdAt'],
-        [sequelize.literal(`"SharingModel"."type"`), 'sharingType'],
-      ],
-      group: [
-        'folder.id',
-        'folder->user.id',
-        'SharingModel.item_id',
-        'sharingType',
       ],
+      group: ['folder.id', 'folder->user.id', 'SharingModel.item_id'],
```

Comment:
Please, take into account that if this is merged before the code in https://github.com/internxt/drive-web/pull/892 is deployed into production, it will break the front as they need `sharingType`. 

I can add this again so it can be merged without breaking the front, but we need to remove `sharingType ` again before moving the front code into prod.